tommath.h

```diff
@@ -703,6 +703,7 @@ MP_DEPRECATED(mp_to_radix) mp_err mp_toradix(const mp_int *a, char *str, int rad
 MP_DEPRECATED(mp_to_radix) mp_err mp_toradix_n(const mp_int *a, char *str, int radix, int maxlen) MP_WUR;
 mp_err mp_to_radix(const mp_int *a, char *str, size_t maxlen, int radix) MP_WUR;
 mp_err mp_radix_size(const mp_int *a, int radix, int *size) MP_WUR;
+mp_err mp_to_decimal(const mp_int *a, char *str, size_t maxlen) MP_WUR;
```

Comment:
I've already asked this before: is there a reason why this has a separate public API function instead of calling it in `mp_to_radix()` when `radix == 10`?

Reply:
I personally think it would be a little nicer to just call it in `mp_to_radix` (though I'll admit that's partly just because then it wouldn't require any changes in MoarVM to take advantage of it (the very reason I made this PR in the first place)). However, I think it was @minad who had an objection/reason.

Reply:
Yep, @minad said that in https://github.com/libtom/libtommath/pull/330#issuecomment-524537046

@minad why do you think we should have separate functions for the different radices instead of all going through `mp_to_radix()` and potentially going to a fast version from there?